### ISSUE TITLE
[ObjectListView_AI4W]: The Name property of a focusable element must not be null

### ISSUE THREAD

**Issue by msftedad** [msftedad/Microsoft---eDAD#2]
User Impact: 
Users who rely on screen readers will face difficulty if the text do not have name property of a focusable elements. 

Environment Details:
OS Version: Version 22H2 (OS Build 22587.1) 
Edge Version: Version 102.0.1245.33 (Official build) (64-bit) 
AT Used: Accessibility Insights for windows. 
Account Used: v-id 

Repro Steps: 
1. Navigate to this link https://sourceforge.net/projects/objectlistview/files/objectlistview/
2. Download the `examples.zip` file present in the link.
3. Open `GettingStarted1.sln` through visual studio and click on the start button to get the demo app running.
4. ObjectListView Getting Started1 will open.
5. Run Accessibility Insights for windows and observe the issue.

Note: 
The same issue repro in other pages as well. 

Actual Result: 
Upon applying Accessibility Insights for windows, text do not have name property of a focusable elements. 

Expected Result: 
Upon applying Accessibility Insights for windows, text should have name property of a focusable elements.  

MAS Rule:[04.02.01: Object Information](https://liquid.microsoft.com/Web/Views/View/470127#Zrex-3A-2F-2Fms-2Eaccessibility-2FRequirements-2F04-2E02-2E01-2F) 
![200759_ObjectiveListView_AI4W](https://user-images.githubusercontent.com/93735775/214543696-32c80875-2bb4-4a25-888f-a808346a24ae.png)

**Comment by msftedad** [comment]
Hi Team, is there any update on this bug?

**Comment by msftedad** [comment]
Hi team, any update on this ticket??
